src/Contexts/section.reducer.jsx

```diff
@@ -92,10 +81,16 @@ export function sectionReducer(state = sectionInitialState, { type, payload }) {
             })
 
         case SECTION_SAVED:
-            state.forEach(section => {
-                if (section._id === payload) section.changed.clear()
+            return state.map(section => {
```

Comment:
Warum werden hier eigentlich immer alle sections behandelt und nicht nur die aktuelle. Das müsste dann in SECTION'S' umbenannt werden.

Reply:
behandelt eigentlich auch nur die aktuelle, theoretisch können zwar auch mehrere sections kurz hintereinander gespiechert worden sein, die triggern das aber alle einzeln und in der regel wird es auch nur die aktuelle sein

Reply:
Dann wäre ein umbennen trotzdem richtig.